card/mod/bootstrap/db/migrate_core_cards/20180423143147_add_type_customized_bootswatch_skin.rb

```diff
@@ -19,18 +19,10 @@ def up
   end
 
   def rename_customized_bootswatch_skin
-    delete_code_card :customized_bootswatch_skin
-    if (card = Card.fetch("Customized bootswatch skin"))
-      if card.codename != "customized_bootswatch_skin"
-        delete_code_card :customized_bootswatch_skin
-      end
-      card.update_attributes! type_id: Card::CardtypeID,
-                              codename: "customized_bootswatch_skin"
-    else
-      ensure_card "Customized bootswatch skin",
-                  type_id: Card::CardtypeID,
-                  codename: "customized_bootswatch_skin"
-    end
+    ensure_card :customized_bootswatch_skin,
+                name: "Customized bootswatch skin",
```

Comment:
if we just do `ensure_card "Customized bootswatch skin",`..., then it works for most decks, yes?  Maybe we should just do that and then fix wikirate by hand?